libevm/lib/refund.go

```diff
@@ -59,3 +62,11 @@ func (s *Service) setStateWithRefund(statedb *state.StateDB, address common.Addr
 	setStateRefunds(statedb, address, key, value)
 	statedb.SetState(address, key, value)
 }
+
+func (s *Service) StateGetRefund(params HandleParams) (error, hexutil.Uint64) {
```

Comment:
Seeems method is not tested.
At the moment we use statedb method directly in the test: `if refund := statedb.GetRefund(); refund != tt.refund {`
Let's use the new method to test the instance properly.